<?xml version="1.0"?>

<launch>
    <!-- Lane Filterer Node -->
    <node pkg="vision" type="lane_filterer.py" name="lane_filter_node" output="screen">
        <!-- HSV Parameters -->
        <param name="higher_h" value="180" />
        <param name="higher_s" value="72" />
        <param name="higher_v" value="255" />
        <param name="lower_h" value="0" />
        <param name="lower_s" value="0" />
        <param name="lower_v" value="157" />

        <!-- Remap topics if needed -->
        <remap from="rgb_image_topic_sub" to="/zed_node/rgb/image_rect_color" />
        <remap from="mask_topic_pub" to="/lane_filter/mask" />
    </node>
</launch>
